ci/azure-pipelines-nightly.yml

```diff
@@ -48,6 +48,25 @@ jobs:
     artifact: opentitan-repo
     displayName: "Upload repository"
 
+- job: hyperdebug
+  displayName: "Hyperdebug tests"
+  timeoutInMinutes: 30
+  dependsOn: checkout
+  pool: FPGA
+  steps:
+  - template: ../ci/checkout-template.yml
+  - template: ../ci/install-package-dependencies.yml
+  - bash: |
+      set -x
+      module load "xilinx/vivado/$(VIVADO_VERSION)"
+      ci/bazelisk.sh test \
+        --define DISABLE_VERILATOR_BUILD=true \
+        --define bitstream=gcp_splice \
+        --build_tests_only \
+        --test_output=errors \
+        $(bash ./bazelisk.sh query 'attr("tags", "[\[ ]hyper310[,\]]", tests(//...))')
```

Comment:
Bazel Q: I see the bob job below is using this kind of query to select tests, but the `rom_e2e` job is using `test_tag_filters` - do these do different things?

Also, should we filter out tests tagged `broken`? (not that there are any right now)